src/shipit_static_analysis/default.nix

```diff
@@ -15,6 +15,38 @@ let
   name = "mozilla-shipit-static-analysis";
   dirname = "shipit_static_analysis";
 
+  mkBot = branch:
+    let
+      cacheKey = "shipit-static-analysis-" + branch;
+      secretsKey = "repo:github.com/mozilla-releng/services:branch:" + branch;
+    in
+      mkTaskclusterHook {
+        name = "Static analysis automated tests";
+        owner = "janx@mozilla.com";
```

Comment:
Nit: Please change this to "jan@", I'm not sure that I actually registered the "janx@" alias.